lib/src/scss-shared/_nx-color-swatches.scss

```diff
@@ -150,3 +150,9 @@ $nx-indigo-300: #bbc5f6;
 $nx-indigo-200: #cad1f3;
 $nx-indigo-100: #d1d6f0;
 $nx-indigo-50: #e7eaf9;
+
+// Black
+$nx-black: #000;
+
+// White
+$nx-white: #fff;
```

Comment:
`$nx-white` and `$nx-black` were already present in `_nx-colors.scss`, so if you add them here you should remove them there.  I would think it wouldn't compile with them present in both places, actually.

Reply:
> `$nx-white` and `$nx-black` were already present in `_nx-colors.scss`, so if you add them here you should remove them there. I would think it wouldn't compile with them present in both places, actually.

Sure it's not problem it just overrides the value of the variable based on load order. Regardless I removed them, the more I thought about it the less value that I saw in having just black and white. https://github.com/sonatype/sonatype-react-shared-components/pull/313/commits/7d76da37609656f5400fcb600c1dd7586dd77efe